.github/CODEOWNERS

```
@@ -0,0 +1 @@
+@bcgov/descw-admin
```

```suggestion
* @bcgov/descw-admin
```
GitHub says this CODEOWNERS file is valid but I think it's supposed to have the `*` so that the user group becomes the code owner of every file in the repo.